### PR TITLE
AB#89414 Add local authority confirm integration test

### PR DESCRIPTION
**Pull request by dawid-pawulski**

### CODE REVIEW

File: src/HE.Investment.AHP.WWW/Views/Site/LocalAuthorityConfirm.cshtml [hunk 1]
```diff
@@ -20,12 +20,12 @@
     @await Html.PartialAsync("~/Partials/Errors/_ErrorSummaryPartial.cshtml")
     <form asp-controller="Site" asp-action="LocalAuthorityConfirm" asp-route-localAuthorityId=@(Model.ViewModel.LocalAuthorityId) asp-route-redirect=@redirect asp-method="post" novalidate>
         <gds-field-set>
-                <gds-h1 size="ControlSize.Xl" class="govuk-!-margin-bottom-6">
-                    @SitePageTitles.LocalAuthorityConfirm
-                </gds-h1>
-                <gds-p>
-                    <b>@Model.ViewModel.LocalAuthorityName</b>
-                </gds-p>
+            <gds-h1 size="ControlSize.Xl" class="govuk-!-margin-bottom-6">
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/Views/Site/LocalAuthorityConfirm.cshtml:23]: Shouldn't page-header be used here?

File: src/HE.Investment.AHP.WWW/Controllers/SiteController.cs [hunk 2]
```diff
@@ -332,7 +337,21 @@ public IActionResult LocalAuthorityConfirm(string siteId, string localAuthorityI
             _mediator,
             new ProvideLocalAuthorityCommand(new SiteId(siteId), localAuthorityId, localAuthorityName, model.Response),
             () => ContinueWithRedirect(new { siteId, redirect }),
-            async () => await Task.FromResult<IActionResult>(View(model)),
+            async () =>
+            {
+                TempData["LocalAuthoritiesErrors"] = ViewBag.validationErrors;
+
+                return await Task.FromResult(RedirectToAction(
+                    "LocalAuthorityConfirm",
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/Controllers/SiteController.cs:345]: please use nameof

**Review by Antek012 — APPROVED** [review]

**Review by Carq — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/Controllers/SiteController.cs (hunk 2 above)
Comment (dawid-pawulski) [reply src/HE.Investment.AHP.WWW/Controllers/SiteController.cs]: code changed

File: src/HE.Investment.AHP.WWW/Views/Site/LocalAuthorityConfirm.cshtml (hunk 1 above)
Comment (dawid-pawulski) [reply src/HE.Investment.AHP.WWW/Views/Site/LocalAuthorityConfirm.cshtml]: done

**Review by jjankowski87 — APPROVED** [review]

**Pull request merged**